docs/src/two-syntax-variants-used

```diff
@@ -0,0 +1,12 @@
+This page uses two different syntax variants: the Cython specific ``cdef`` syntax
+and static Cython type declarations in pure Python code,
+following `PEP-484 <https://www.python.org/dev/peps/pep-0484/>`_ type hints
+and `PEP 526 <https://www.python.org/dev/peps/pep-0526/>`_ variable annotations.
+To make good use of the latter, including C data types etc., you need
+the special ``cython`` module, which you can import with
+
+.. code-block:: python
+
+    import cython
+
+in the Python module that you want to compile.
```

Comment:
I still stand by having an end-note mentioning [Pure Python Mode](https://cython.readthedocs.io/en/latest/src/tutorial/pure.html).

Reply:
I already added a link to the line mentioning `pure Python code` above at some point. Looks like that was lost.

Reply:
@scoder can you add it again?

Reply:
in this case, I suppose, make an suggestion.

Reply:
Found it. I had added it to the suggestion in the ticket:
https://github.com/cython/cython/issues/4187

Reply:
> Found it. I had added it to the suggestion in the ticket:
> #4187

Yeah, okay, I haven't followed the updates vigilantly enough.

Reply:
resolved [here](https://github.com/cython/cython/pull/4226/commits/1ef31d142b220d0da8ea097f6b90fbad0ccc54bb)